### DISCUSSION TITLE
Example of creating a feature

### DISCUSSION THREAD

**Discussion by MincePie** [MichalLytek/typegraphql-prisma#256] [Q&A]
Do you know of a tutorial demonstrating how to create a feature, starting with the enum and model in the schema.prisma, and going right through to the form to submit the mutation?

I added a model and an enum to the schema file and ran generate, expecting the graphql.tsx to be generated with all the crud resolvers. Instead, I get types and enums but nothing to interact with the db. I created a resolver, but don't know how to work with the idea behind this tool to have it available in grapqhql.tsx.

I'm using this boilerplate repo as a starting point: https://github.com/NoQuarterTeam/boilerplate. I'd love to figure out how to use it to create a feature on both the back and front ends.

Do you know of any more extensive tutorials/examples that step through each step of the process? Thank you.

**Comment by MichalLytek** [comment]
https://prisma.typegraphql.com/docs/intro

**Comment by MincePie** [comment]
Thanks. I've been through those docs many times. Still cant find an example of how to use it. I have tried several tutors to this point and am still looking for someone that can work this through. I'll share if I ever figure it out.

**Reply by MichalLytek** [reply]
1) Create enum and model in Prisma
2) Run `npx prisma generate`
3) Import the generated resolvers (or if you use `resolvers`, all the new ones are automatically included)
4) No more steps, if you done the introduction/bootstrap phase properly.

**Comment by MincePie** [comment]
Well, the resolvers didn't get created. I'm looking for a tutorial to step through this in greater detail whilst I continue to search for a tutor that can help to make sense of this. Thanks anyway. I'll share if I figure out how to to use this.

**Reply by MichalLytek** [reply]
Please create a reproducible code example repository. I can't guess what you did wrong. I need to see the config or code.

**Comment by MincePie** [comment]
I just added you. I tried to make a Request resolver (in api/src/modules/conciergedesk/conciergeDeskResolver.ts. I can't find what I need to do to get that resolver to become available in web/src/lib/graphql.tsx. The types got generated but the mutation did not. Thank you very much.

**Reply by MichalLytek** [reply]
`output   = "../../../../node_modules/@generated"`

You're looking for mutations in frontend folder generated by `graphql-codegen` while prisma generated resolvers are on backend in node_modules.

Sorry, you need to step back and learn the basics. I don't have enough time to help all newbies with basic stuff. I'm sorry.

Do a step back and create a mini app by yourself, not try to edit complex big boilerplates you don't understand.

**Comment by MincePie** [comment]
No problem. The resolver hasn't been created anywhere in the project file. Note that the user mutations did get created as resolvers in that same file I referenced. Thanks anyway. Ill keep trying to find a way to figure out what needs to happen. If I figure it out, I'll share a note for others that might be struggling to make sense of the logic.
